web/themes/custom/contribtracker/ts/contrib-graph.js

```diff
@@ -0,0 +1,94 @@
+(function ($) {
+  jQuery(function () {
```

Comment:
I don't understand this. Are we, by any chance, creating a jQuery plugin here? Would it be too complicated to write this using Drupal behaviors?